src/services/components/getComponents/handler.ts

```diff
@@ -0,0 +1,21 @@
+import { PrismaClient } from '@prisma/client';
+import { middyfy } from '@lib/middleware';
+import { getComponents } from './useCase';
+import createError from 'http-errors';
+
+const prisma = new PrismaClient();
+
+export default middyfy(async (event) => {
+  try {
+    console.log('Received CloudFormation Event:', JSON.stringify(event, null, 2));
+
+    // UseCase - Get Components
+    const result = await getComponents(prisma);
```

Comment:
What about if we have 10k components, are we gonna return them all?

Reply:
were planning on implementing limiting features to get only a set amount at a time